manuscript/book.md

```diff
@@ -2370,7 +2370,7 @@ expect(getMessageProps(1, 5, 0, 2, 0)).toEqual([
 ])
 -->
 
-But this makes the function API less discoverable and makes editor autocomplete less useful. It also makes the impression that the function accepts any number of arguments, not strictly five, which was clear in the previous iteration.
+But this makes the function API less discoverable and can make editor autocomplete less useful. It also gives the impression that the function accepts any number of arguments - not strictly five - which was clear in the previous iteration.
 
```

Comment:
Also, the order of the `MESSAGE_IDS` must remain bound to the order of the `counts` that are passed in... this becomes more difficult to see when applying the spread operator.

Reply:
Added!